set.mm

```diff
@@ -531652,17 +531554,17 @@ written as (is equal to) a class abstraction does not require these
        that the only $a-statements with typecode ` class ` are ~ cv , ~ cab and
        statements corresponding to defined class constructors.
 
-       UPDATE:  This theorem is (almost) ~ abid2 and ~ bj-abid2 , though the
-       present proof is shorter than a proof from ~ bj-abid2 and ~ eqcomi (and
-       is shorter than the proof of either); plus, it is of the same form as
-       ~ cvjust and such a basic statement deserves to be present in both
-       forms.  Note that ~ bj-termab shortens the proof of ~ abid2 , and
-       shortens five proofs by a total of 72 bytes.  Move it to Main as "abid1"
-       proved from ~ abbi2i ?  Note also that this is the form in Quine, more
-       than ~ abid2 .  (Contributed by BJ, 21-Oct-2019.)
+       UPDATE:  This theorem is (almost) ~ abid2 , though the present proof is
+       shorter than a proof from ~ abid2 and ~ eqcomi (and is shorter than the
+       proof of either); plus, it is of the same form as ~ cvjust and such a
+       basic statement deserves to be present in both forms.  Note that
+       ~ bj-termab shortens the proof of ~ abid2 , and shortens five proofs by
+       a total of 72 bytes.  Move it to Main as "abid1" proved from ~ abbi2i ?
+       Note also that this is the form in Quine, more than ~ abid2 .
+       (Contributed by BJ, 21-Oct-2019.)
        (Proof modification is discouraged.) $)
     bj-termab $p |- A = { x | x e. A } $=
-      ( cv wcel biid bj-abbi2i ) ACBDZABGEF $.
+      ( cv wcel biid abbi2i ) ACBDZABGEF $.
```

Comment:
incidentally this is identical to abid1

Reply:
Thanks: this was actually moved to Main as ~ abid1 as proposed, but I forgot to remove it here. Fixed.